packages/playwright-test/src/common/fixtures.ts

```diff
@@ -239,7 +239,12 @@ function innerFixtureParameterNames(fn: Function, location: Location, onError: L
     onError({ message: 'First argument must use the object destructuring pattern: '  + firstParam, location });
     return [];
   }
-  const props = splitByComma(firstParam.substring(1, firstParam.length - 1)).map(prop => {
+  const firstParamWithoutBrackets = firstParam.substring(1, firstParam.length - 1);
+  if (firstParamWithoutBrackets.includes('...')) {
+    onError({ message: 'Fixtures cannot use rest parameters: ' + firstParamWithoutBrackets.trim(), location });
```

Comment:
It is called "rest property": https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#rest_property.